assets/js/base/context/providers/cart-checkout/checkout-processor.js

```diff
@@ -58,7 +58,8 @@ const CheckoutProcessor = () => {
 		paymentMethods,
 		shouldSavePayment,
 	} = usePaymentMethodDataContext();
-	const { addErrorNotice, removeNotice, setIsSuppressed } = useStoreNotices();
+	const { setIsSuppressed } = useStoreNoticesContext();
```

Comment:
The reason is:

1) This is not a value on the `core/notices` store. This is a store created in Gutenberg and we don't control what shape the state gets. I don't think this is something we could make a PR to Gutenberg for as it is something specific to our use-case and would not add value to Gutenberg.
2) Its value is `true` if the express payment method is active and `false` otherwise, so we can derive this value from the payment method data store when this the work to convert that is done. We need to create a follow up issue (but I think GitHub will do this automagically based on the TODO comment!)